File: employeeservice/src/main/java/com/ebf/employeeservice/model/Employee.java
package com.ebf.employeeservice.model;

import javax.persistence.Entity;
import javax.persistence.FetchType;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.JoinColumn;
import javax.persistence.ManyToOne;
import javax.validation.constraints.NotEmpty;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
public class Employee {

  @Id
  @GeneratedValue
  private long id;
  private String name;
  private String surname;
  private String email;
  private String address;
  private double salary;
  @ManyToOne(fetch = FetchType.LAZY, optional = false)
  @JoinColumn(name = "company_id", referencedColumnName = "id")
  private Company company;

}

File: employee_frontend/app/models/employee.js
import Model, { attr } from '@ember-data/model';

export default class EmployeeModel extends Model {
  @attr('string') name;
  @attr('string') surname;
  @attr('string') email;
  @attr('string') address;
  @attr('string') salary;
  @attr('number') companyId;
}
